x-pack/plugins/secops/server/graphql/sources/sources.mock.ts

```diff
@@ -6,7 +6,10 @@
 
 /* tslint:disable */
 
-export default
+import { Logger } from '../../utils/logger';
+import { Context } from '../index';
+
+export const sourcesDataMock =
```

Comment:
So, the thinking of why I put `tslint:disable` to this file is that this file contains just raw JSON based data for mocks since they can be very large at some points in time and we have an easier time copying them since we don't have a "record of data" capability just yet.

So this is just some background. I think that putting the `getAllSourcesQueryMock` together with this file is still ok, but we might not get good linting since we are turning it off for the potentially large big glob of JSON type data. Maybe if we can get away with small mocks, we can turn off linting and just manually edit the files and put up with that pain for now?

I'm up for either....but having the tslint disabled with regular code below might get messy since it turns off everything for the whole file.